components/elm/src/dyn_subgrid/dynHarvestMod.F90

```diff
@@ -66,22 +65,23 @@ module dynHarvestMod
   ! for FATES: capacity for passing harvest data in units of carbon harvested per year (per grid cell) has been added
   ! but these data are not yet included in the input file
   ! the code here can be changed to wood_harvest_units = harvest_carbon to pass carbon data to FATES if:
-  !  the carbon data are in the same input variables as listed below
-  !  and the carbon units in the input file match that expected by FATES
+
+  ! the carbon data are in the same input variables as listed below
+  ! and the carbon units in the input file match that expected by FATES
 
   integer, public, parameter :: num_harvest_vars = 5
   character(len=64), public, parameter :: harvest_varnames(num_harvest_vars) = &
-       [character(len=64)  :: 'HARVEST_VH1', 'HARVEST_VH2', 'HARVEST_SH1', 'HARVEST_SH2', 'HARVEST_SH3']
+       [character(len=64) :: 'HARVEST_VH1', 'HARVEST_VH2', 'HARVEST_SH1', 'HARVEST_SH2', 'HARVEST_SH3']
+
+  type(dyn_var_time_uninterp_type) :: harvest_vars(num_harvest_vars)   ! value of each harvest variable
+
   ! the units flag must match the units of harvest_varnames
   ! set this here because dynHarvest_init is called after alm_fates%init
   ! this flag is accessed only if namelist do_harvest is TRUE
-  integer, public          :: wood_harvest_units = hlm_harvest_area_fraction
-  
-  type(dyn_var_time_uninterp_type) :: harvest_vars(num_harvest_vars)   ! value of each harvest variable
-
-  real(r8) , allocatable, public   :: harvest_rates(:,:) ! category harvest rates (d1) in each gridcell (d2)
 
-  logical, private         :: do_harvest ! whether we're in a period when we should do harvest
+  integer, public, parameter    :: wood_harvest_units = 2    ! 1 = area fraction, 2 = carbon
```

Comment:
@rgknox the default used to be the area fraction option (i.e. `hlm_harvest_area_fraction`).  Should this be reinstated as the default?

Reply:
I defer to @aldivi @jenniferholm and @sshu3

Reply:
Sorry for the confusion @glemieux. This parameter shall be changed to 1. Currently merged works do not include carbon-based harvest yet.

Reply:
@rgknox - apologies for my delay in responding to this question.... but I see that Shijie has answered it, and that the flagged should be turned back to 1.